conformance/02_push_test.go

```diff
@@ -330,21 +330,28 @@ var test02Push = func() {
 		})
 
 		g.Context("Teardown", func() {
+			deleteManifest := func(req *reggie.Request) {
+				SkipIfDisabled(push)
+				RunOnlyIf(runPushSetup)
+				resp, err := client.Do(req)
+				Expect(err).To(BeNil())
+				Expect(resp.StatusCode()).To(SatisfyAny(
+					SatisfyAll(
+						BeNumerically(">=", 200),
+						BeNumerically("<", 300),
+					),
+					Equal(http.StatusMethodNotAllowed),
+				))
+			}
+			deleteManifests := func() {
+				deleteManifest(client.NewRequest(reggie.DELETE, "/v2/<name>/manifests/<digest>",
+					reggie.WithDigest(manifests[1].Digest)))
+				deleteManifest(client.NewRequest(reggie.DELETE, "/v2/<name>/manifests/<digest>",
+					reggie.WithDigest(emptyLayerManifestDigest)))
```

Comment:
We need a condition on this second delete, checking whether `skipEmptyLayerTest` is false before running it.

Reply:
See #421 for while this will be a bigger issue going forward.

Reply:
Have fixed this in #421 since hitting this issue there as well.